redis/base.cfg

```diff
@@ -0,0 +1,81 @@
+# Redis base configuration.
+# Use development.cfg or production.cfg, not base.cfg directly.
+
+[redis-settings]
+# port = # set by development.cfg or production.cfg
+# redis-version = # set by [version].cfg
+bind = 127.0.0.1
```

Comment:
Since we are using the default config, this also applies:
```
# By default protected mode is enabled. You should disable it only if
# you are sure you want clients from other hosts to connect to Redis
# even if no authentication is configured, nor a specific set of interfaces
# are explicitly listed using the "bind" directive.
protected-mode yes
```

---

In my opinion it is important to always use the standard configuration of the redis-version used, and only configure what's needed. Therefore
- I chose not to create a config template and render it with our settings, since the template will be outdated.
- I chose to use the default settings and overwrite the settings with `src/redis-server --port xx`-like parameters; thus we always have (hopefully) sane defaults regarding security as long as we do not influence it
- I explicitly made the download recipe unpack to e.g. `parts/redis-3.2.3`, so that we can just change the redis version and it will download it to a new directory and therefore use the new standard config, which is hopefully better but still compatible 😉